products/opensuse/main.pm

```diff
@@ -347,6 +347,9 @@ elsif (get_var("SUPPORT_SERVER")) {
     loadtest "support_server/boot";
     loadtest "support_server/login";
     loadtest "support_server/setup";
+    loadtest "support_server/meddle_multipaths" if (get_var("SUPPORT_SERVER_TEST_INSTDISK_MULTIPATH"));
```

Comment:
As this looks unrelated to the motivation of the change mentioned of PR, could you please move it to the separate commit?

Reply:
Fair enough. I'll comply in a PR amendment. This means, though, that the Tumbleweed verification run demanded by another reviewer is no longer possible here, but only after both commits.

Reply:
As required, submitted for opensuse: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9783